Test2Roulette

```diff
@@ -0,0 +1,404 @@
+#include <iostream>
+#include <clocale>
+#include <ctime>
+using namespace std;
+
+//Konstanter
+//Max och minimum värden för slump-funktion
+int const MAX_VALUE = 36;
+int const MIN_VALUE = 1;
+//Minsta tillåtna insats
+int const MIN_BET = 100;
+//Faktor att multiplicera insats med vid vinst vid rätt färg respektive nummer
+int const COLOR_PAYOUT = 2;
+int const NUMBER_PAYOUT = 10; 
+
+//Global struct för hjulets utfall; nummer och färg.
+struct WheelOutcome
+{
+    int number;
+    char color;
+};
+
+//Globala variabler
+bool gameOver;
+int moneyRemaining;
+int totMoneyWon;
+int numPlayedRounds = 0;
+int currentBet;
+int gameMode;
+int numPicked;
+char colPicked;
+WheelOutcome wheelOutcome;
+
+
+//Deklarerar funktioner
+void StartUp();
+void NewRound();
+void PlaceBet();
+void ValidateBet();
+void PickGameMode();
+void PickNumOrCol();
+void SpinWheel();
+void CalculateColor();
+void CalculateOutcome();
+void EmptyNewLines();
+
+
+int main()
+{
+    //Möjliggör stöd för åäö i konsollfönstret
+    setlocale(LC_ALL, "swedish");
+    
+    //Kör uppstartsfunktion
+    StartUp();
+
+    //En while-loop som kör så länge spelaren inte väljer att avsluta eller har för lite pengar på kontot
+    while (!gameOver)
+    {
+        //Funktioner att anropa under spelets gång
+        NewRound();
+        //If-sats som kontrollerar om spelaren valt att avsluta inför ny runda
+        if (gameOver)
+        return 0;
+        PlaceBet();
+        PickGameMode();
+        PickNumOrCol();
+        SpinWheel();
+        CalculateColor();
+        CalculateOutcome();
+    }
+
+}
+
+void StartUp()
+{
+    //Initierar globala variabler för pågående spel och återstående pengar
+    gameOver = false;
+    moneyRemaining = 1000;
+    char answer;
+    int age;
+    cout << "Välkommen till Fabians roulettehjul!\n";
+    cout << "Du måste vara över 18 år för att spela.\n";
+    //Kontrollera spelarens ålder
+    do
+    {
+        cout << "Ange din ålder: \n";
+        cin >> age;
+        cin.clear();
+        cin.ignore(10000, '\n');
```

Comment:
Dessa raderna upprepas flera gånger kunde man møjligtvis stoppa in i egen funktion skriver mer om det på ett bættre exempel